t/0700-unifyfs-stage-full.t

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Test unifyfs-stage executable for basic functionality
+#
+
+test_description="Test basic functionality of unifyfs-stage executable"
+
+. $(dirname $0)/sharness.sh
+
+#test_expect_success "Intercepted mount point $UNIFYFS_MOUNTPOINT is empty" '
+#    test_dir_is_empty $UNIFYFS_MOUNTPOINT
+#'
```

Comment:
Can remove this. Assuming it's leftover from copying the file from a previous test.

Reply:
Oops.  I thought I'd cleaned out all the cruft; looks like I missed a few.

Reply:
removed in patch.